README.md

```diff
@@ -40,19 +40,54 @@ should look something like this:
 
 ## Configuration
 
-After booting the SonarQube Server with the License-Check Plugin two new options can be found in the tab
-<b>Administration</b>.
+## General Configuration
+After booting the SonarQube Server with the License-Check Plugin be found in the tab <b>Administration</b>.
+
+* Within the <b>General Settings</b> and <b>License Check</b> you find the settings for the plugin.
```

Comment:
Maybe mention that the version under "License Check" is just a nicer user interface for same settings under "General Settings"

Reply:
ok - I can also do some sreenshots for this case